cmd/acikubectl/cmd/debug.go

```diff
@@ -515,12 +558,47 @@ func accVersionCmdArgs(systemNamespace string) []string {
 		"aci-containers-controller", "--version"}
 }
 
+func controllerConfigCmdArgs(systemNamespace string) []string {
+	return []string{"-n", systemNamespace, "exec", "deployment/aci-containers-controller",
+		"-c", "aci-containers-controller", "--", "curl", "-s", "localhost:8091/config"}
+}
+
+func controllerStatusCmdArgs(systemNamespace string) []string {
+	return []string{"-n", systemNamespace, "exec", "deployment/aci-containers-controller",
+		"-c", "aci-containers-controller", "--", "curl", "-s", "localhost:8091/status"}
+}
+
 func aciContainerHostVersionCmdArgs(systemNamespace string) []string {
 	return []string{"-n", systemNamespace, "exec", "daemonsets/aci-containers-host",
 		"-c", "aci-containers-host", "-i", "--",
 		"aci-containers-host-agent", "--version"}
 }
 
+func hostIpamLogCmdArgs(systemNamespace string, podName string, containerName string, args []string) []string {
```

Comment:
Can we combine the 5 host agent functions, since they are taking the same arguments and the way you have designed it should be able to handle modifications to the individual commands in future.